### PR TITLE
finished 1.2.7 1.2.8

### PR DESCRIPTION
**Pull request by mikechengwei**

### CODE REVIEW

**Review by tennix — APPROVED** [review]
LGTM

**Review by cofyc — APPROVED** [review]
LGTM

**Comment by cofyc** [comment]
ah, can you take a look at this failure: https://travis-ci.org/pingcap-incubator/tidb-in-action/jobs/659539494?utm_medium=notification&utm_source=github_status

```
error: error while generating page "session2/chapter1/tidb-operator-tools.md": 
Template render error: (/home/travis/build/pingcap-incubator/tidb-in-action/session2/chapter1/tidb-operator-tools.md) [Line 49, Column 9]
  unexpected token: <
The command "gitbook build ." exited with 1.
cache.2
store build cache

```

`gitbook build` fails on your submitted file `session2/chapter1/tidb-operator-tools.md`.

File: session2/chapter1/tidb-operator-deployment-private-env.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 2. Kubernetes v1.12.5+
 3. CentOS 7.6 以上，内核要求为 3.10.0-957 或之后版本
 
 ### 部署 Cloud TiDB 资源要求如下:
+
 1. 至少 3 台服务器可供使用，服务器采用万兆网卡
 2. 至少 6 个 PV 可供使用(3 PV 用于 TiKV 实例，分布于不同服务器；3 PV用于 PD 部署，分布于不同服务器)
 
 ### Linux 内核参数要求如下:
+
```
Comment (siddontang) [inline session2/chapter1/tidb-operator-deployment-private-env.md]: 内核这块用 code block 来 wrap 下

**Comment by cofyc** [comment]
@mikechengwei hi, can you resolve the conflicts in `tidb-operator-deployment-private-env.md`?

**Comment by cofyc** [comment]
you can revert the changes in `tidb-operator-deployment-private-env.md` then we can merge your submitted files first.

**Comment by mikechengwei** [comment]
OK wait for a moment

**Comment by mikechengwei** [comment]
@cofyc help me merge this pr ,thanks

**Review by cofyc — APPROVED** [review]

**Comment by winkyao** [comment]
@mikechengwei Thanks for your contribution.

**Pull request merged**